build.gradle

```diff
@@ -732,6 +742,10 @@ ext.libraries = [
             exclude(group: 'org.slf4j', module: 'slf4j-api')
             force = true
         },
+        spockframework   : [
```

Comment:
Didn't know if you wanted to pollute your other test libraries with Spock. If that would be fine, I can move.